contracts/GenesisSupply.sol

```diff
@@ -172,6 +176,25 @@ contract GenesisSupply is AccessControl {
         return tokenId;
     }
 
+    /**
+     * Mint reserved gods
+     * This function needs to be ran BEFORE the mint is opened to avoid
+     * @param count number of gods to transfer
+     */
+    function mintReservedGods(uint256 count) public onlyRole(GENESIS_ROLE) {
+        uint256 nextIndex = reservedGodsCurrentIndex();
+        // Here we don't need to increment counter and god supply counter because we already do in the constructor
+        // to not initialize the counters at 0
+        for (uint256 i = nextIndex; i < count + nextIndex; i++) {
+            reservedGodsTransfered.increment();
```

Comment:
Shouldn't `tokenCounter` be incremented as well?

Reply:
no because in the constructor we already increment this to 10 initially. Kinda shitty design IMO, but we save on gas on construction when counters start at something other than 0. The comment explains it, but yea, not super clear.